src/org/labkey/test/tests/TriggerScriptTest.java

```diff
@@ -679,23 +679,29 @@ private void insertSingleRowViaUI(String dataRegionName, Map<String,String> reco
 
     /**
      * delete single record via the table UI
-     * @param columnName Column to look at
-     * @param columnValue value to look for
-     * @param tableName DataRegionTable name
+     *
+     * @param columnName       Column to look at
+     * @param columnValue      value to look for
+     * @param tableName        DataRegionTable name
+     * @param deleteButtonText text that appears in delete confirmation when not in an alert.
+     * @param expectPageLoad indicates whether confirming deletion will result in a page load or not
      */
-    private void deleteSingleRowViaUI(String columnName, String columnValue, String tableName)
+    private void deleteSingleRowViaUI(String columnName, String columnValue, String tableName, String deleteButtonText, boolean expectPageLoad)
     {
         DataRegionTable drt = new DataRegionTable(tableName, this);
         int rowId = drt.getRowIndex(columnName, columnValue);
         drt.checkCheckbox(rowId);
-        doAndWaitForPageToLoad(() ->
+        doAndMaybeWaitForPageToLoad(1000, () ->
```

Comment:
One second is a little short for a page load. `defaultWaitForPage` is the standard duration.

Reply:
Done.